src/psim/simulations/dual_orbit.cpp

```diff
@@ -47,6 +49,11 @@ DualOrbitGnc::DualOrbitGnc(
   add<SatelliteTruthNoAttitudeGnc>(randoms, config, "follower");
   add<HillFrameEci>(randoms, config, "leader", "follower");
   add<HillFrameEci>(randoms, config, "follower", "leader");
+  add<NormVector3>(randoms, config, "truth.leader.hill.dr");
+  add<NormVector3>(randoms, config, "truth.leader.hill.dv");
+
+
+
   // Sensors model
```

Comment:
Let's clean up these extra blank lines here:
```suggestion
  add<NormVector3>(randoms, config, "truth.leader.hill.dv");
  // Sensors model
```